src/popup/helper.js

```diff
@@ -112,3 +112,38 @@ export const licenseInfo = {
     licenseDescription: 'This work has been marked as dedicated to the public domain.',
   },
 };
+
+export const tooltipInfo = {
+  by: {
+    tooltipHeading: `License CC BY`,
+    tooltipContent: `<p><i class='icon cc-by'></i> Credit the creator.</p>`,
+  },
+  bync: {
+    tooltipHeading: `License CC BY-NC`,
+    tooltipContent: `<p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-nc'></i> Noncommercial uses only.</p>`,
+  },
+  bysa: {
+    tooltipHeading: `License CC BY-SA`,
+    tooltipContent: `<p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-sa'></i> Share adaptations under the same terms.</p>`,
+  },
+  bynd: {
+    tooltipHeading: `License CC BY-ND`,
+    tooltipContent: `<p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-nd'></i> No derivatives or adaptations permitted.</p>`,
```

Comment:
As already said multiple times, the information is already present in `licenseInfo` object. There is no need to create a new one.

Reply:
i would be working on the things as mentioned.